cpp/src/barretenberg/plonk/composer/ultra_composer.test.cpp

```diff
@@ -793,4 +793,25 @@ TYPED_TEST(ultra_composer, range_checks_on_duplicates)
     TestFixture::prove_and_verify(composer, /*expected_result=*/true);
 }
 
+TEST(ultra_composer, range_constraint_problem)
```

Comment:
This can be either better documented here or made another PR.

Reply:
Tests are often the best documentation, so I'd say leave it in and add some comments explaining the issue. One step further, I would implement the test using a lambda `run_test(bool add_a_to_execution_trace, bool add_b_to_execution_trace, bool add_c_to_execution_trace)` and illustrate what happens when various combinations of gates are created.

Reply:
I added a TODO because some of the details of the issue are still unclear, is that fine?